test/e2e/cloud/gcp/auth/service_account_admission_controller_migration.go

```diff
@@ -46,8 +46,7 @@ var _ = SIGDescribe("ServiceAccount admission controller migration [Feature:Boun
 			}
 			testSuite.TestCases = append(testSuite.TestCases, serviceaccountAdmissionControllerMigrationTest)
 
-			extraEnvs := []string{"KUBE_FEATURE_GATES=BoundServiceAccountTokenVolume=true"}
```

Comment:
what is this test exercising now that 1.21 and 1.22 both have the feature enabled?

Reply:
https://github.com/kubernetes/test-infra/blob/36dd89144e5c02b40746f4199d7bf0b6d7c454db/config/jobs/kubernetes/sig-auth/serviceaccount-admission-controller-migration-config.yaml#L25-L26

it is testing 1.20 -> latest (1.22)